Makefile

```diff
@@ -326,6 +326,11 @@ lint:
 lint-metrics:
 	./hack/prom_metric_linter.sh  --operator-name="kubevirt" --sub-operator-name="ssp"
 
+.PHONY: lint-monitoring
+lint-monitoring:
+	go install github.com/kubevirt/monitoring/monitoringlinter/cmd/monitoringlinter@e2be790
```

Comment:
Please install it to the `$(LOCALBIN)` directory. Similarly to how `controller-gen` is installed on line 250.

As a future task, can you create a release tag in the `github.com/kubevirt/monitoring` repo for the submodule `monitoringlinter`? Then we can use a released version instead of a git commit `e2be790`.

Reply:
Updated to use `$(LOCALBIN)`, thanks.

We did create a release tag in https://github.com/kubevirt/monitoring/releases, but there is an issue with installing it using `v0.0.5` tag, probably because it's a submodule:
```
 go install github.com/kubevirt/monitoring/monitoringlinter/cmd/monitoringlinter@v0.0.5
go: github.com/kubevirt/monitoring/monitoringlinter/cmd/monitoringlinter@v0.0.5: module github.com/kubevirt/monitoring@v0.0.5 found, but does not contain package github.com/kubevirt/monitoring/monitoringlinter/cmd/monitoringlinter
```
We decided to use commit hash because of that issue, and also because this way we don't need to release all `kubevirt/monitoring` repo for changes in the linter.

Reply:
Yes, you need to create a separate tag for submodule. Like we do in `ssp-operator` for the `api` submoule: https://github.com/kubevirt/ssp-operator/tags